### ISSUE TITLE
ngx-extended-pdf-viewer causes window.print() not working

### ISSUE THREAD

**Issue by zhenhaolin** [stephanrauh/ngx-extended-pdf-viewer#101]
I'm having a similar problem with that post:
https://stackoverflow.com/questions/56711126/window-print-throws-me-an-program-stopping-error/56719053

and after using the answer to this post I fixed the window.print() function by removing these three JavaScript files from the "scripts" section and the translation from the "assets" section:
```json
"scripts": [
          "node_modules/ngx-extended-pdf-viewer/assets/pdf.js",
          "node_modules/ngx-extended-pdf-viewer/assets/pdf.worker.js",
          "node_modules/ngx-extended-pdf-viewer/assets/viewer.js"
        ]

"assets": [
          "src/favicon.ico",
          "src/assets",
          {
            "glob": "**/*",
            "input": "node_modules/ngx-extended-pdf-viewer/assets/locale",
            "output": "/assets/locale/"
          }
        ]
```
which causes the pdf-viewer not working, but I still want to use ngx-extended-pdf-viewer for my other pages, so how can I do that.

the package version is : "ngx-extended-pdf-viewer": "^0.9.25",

**Comment by zhenhaolin** [comment]
![image](https://user-images.githubusercontent.com/31851019/60656357-93282f00-9e4f-11e9-968b-71c0e29a167d.png)

**Comment by stephanrauh** [comment]
Having an omelette without breaking the egg is a high art. Essentially, that's what you're asking for, giving that the viewer of pdf.js itself overrides the standard JavaScript ´print()` function.

However, I don't see why they do it, so I've decided to do something about it. But I don't claim to be more clever than the pdf.js team, so let's play this safely.

I've published a new version, 0.9.53, containing a bugfix. **Caveat:** I've implemented the bugfix exclusively in the developer preview files (pdf-2.2.222.js, pdf-worker-2.2.222.js, and viewer-2.2.222.js). You'll have to modify you `angular.json` file to see the difference. Another difference is that these files aren't transpiled to ES5, so maybe they aren't compatible to IE11 (I didn't check that yet).

@zhenhaolin Please update to ngx-extended-pdf-viewer 0.9.53, import the 2.2.222 files instead of the standard files, and run a test. I'm positive the bug you've reported is solved. What I do not know is if there are side effects. What happens if you're hitting the print button on the Windows keyboard? Things like that.

If everything turns out to be fine, I'll happily apply the bugfix to the standard files 2.1 files (the files without version number).

Thanks in advance,
Stephan

**Comment by stephanrauh** [comment]
Also see https://github.com/mozilla/pdf.js/issues/10944

**Comment by stephanrauh** [comment]
@timvandermeij reports they've most like overwritten the window.print() function to support the standard key bindings Ctrl + P and the print button.

**Comment by stephanrauh** [comment]
I've just published version 0.9.54. It should solve your bug. 

There's still an issue with the overlay manager, but I doubt any user even notices. If you've navigated to a pdf.js page, than away, and then return to another pdf.js page, the print overlay is broken. The PDF file in the background doesn't get grayed out. I'll move that to another ticket.
